pkgs/by-name/gi/git-gamble/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitLab,
+  git,
+  installShellFiles,
+  nix-update-script,
+}:
+
+let
+  version = "2.9.0";
+
+  src = fetchFromGitLab {
+    owner = "pinage404";
+    repo = "git-gamble";
+    rev = "version/${version}";
+    hash = "sha256-hMP5mBKXcO+Ws04G3OxdYuB5JoaSjlYtlkerRQ6+bXw=";
+  };
+in
+rustPlatform.buildRustPackage {
+  pname = "git-gamble";
+  inherit version src;
+
+  cargoHash = "sha256-vrzcNdLY2PkyZ1eLwOiONRHVAolbTDxytEgi09WkDZQ=";
+
+  nativeCheckInputs = [ git ];
+  preCheck = ''
+    patchShebangs tests/editor/fake_editor.sh
+  '';
+
+  nativeBuildInputs = [ installShellFiles ];
```

Comment:
at runtime, `git-gamble` needs to have `git` in the `$PATH`

i tryied to add `buildInputs = [ git ];`

but that don't make `git` available in the `$PATH` when installing `git-gamble`, i didn't see the difference, so i removed the line

Reply:
Would wrapping `git-gamble` in `postInstall` make `git` available to `git-gamble` at runtime, @pinage404 ?

```nix
    wrapProgram git-gamble \
      --prefix PATH : "${lib.makeBinPath [ git ]}"
```

Reply:
it works, thanks !

i see `result/bin/git-gamble` but i don't see `result/bin/git`

Without this direct symbolic link, is it possible for the garbage collector to delete `git` if the user doesn't depend on it directly ?